### DISCUSSION TITLE
Way to sponsor the project?

### DISCUSSION THREAD

**Discussion by philstopford** [CineEncoder/cine-encoder#40] [Q&A] [answered]
It looks like PayPal isn't allowing me to send donations, for some unspecified reason (but I guess it has to do with Ukraine). Is there an alternative way to provide support?

**Comment by CineEncoder** [comment, marked as answer]
Hello! 
Thank you for your willingness to help develop the project. When a new version of the program is released, I will update the link to the new PayPal account on the main page of the project and in the application in the Donate section.
